src/NUnitFramework/framework/TestContext.cs

```diff
@@ -75,6 +76,18 @@ public static TextWriter Out
             get { return TestExecutionContext.CurrentContext.OutWriter; }
         }
 
+#if !NETCF && !SILVERLIGHT && !PORTABLE
+        /// <summary>
+        /// Gets a TextWriter that will send output directly to Console.Error
+        /// </summary>
+        public static TextWriter Error = new EventListenerTextWriter("Error", Console.Error);
+
+        /// <summary>
+        /// Gets a TextWriter for use in displaying immediate progress messages
+        /// </summary>
+        public static readonly TextWriter Progress = new EventListenerTextWriter("Progress", Console.Error);
```

Comment:
Even if I say it myself, I think it's rather cool how the console is handling this output. There is a test called TextOutputTests.MultipleWrites that displays various kinds of output on the console. If you run it you can see how the output combines. All the labels get combined so long as no parallel-running test produces output in the middle. Each line is colored as needed.